# HG changeset patch
# User Victor Norton <vic@norton.name>
# Date 1489328512 0
#      Sun Mar 12 14:21:52 2017 +0000
# Branch stable
# Node ID 0cf53cd75e13eb4d66755c699cb05aad4f1e0c38
# Parent  a21f8372bb76224b2bcf2d94151bfb730b259716
doc: Fix eig output argument description (bug #50524).

* eig.cc (Feig): Fix eig output argument description.

diff --git a/libinterp/corefcn/eig.cc b/libinterp/corefcn/eig.cc
--- a/libinterp/corefcn/eig.cc
+++ b/libinterp/corefcn/eig.cc
@@ -40,18 +40,19 @@ DEFUN (eig, args, nargout,
 @deftypefnx {} {@var{lambda} =} eig (@var{A}, @var{B})
 @deftypefnx {} {[@var{V}, @var{lambda}] =} eig (@var{A})
 @deftypefnx {} {[@var{V}, @var{lambda}] =} eig (@var{A}, @var{B})
 @deftypefnx {} {[@var{V}, @var{lambda}, @var{W}] =} eig (@var{A})
 @deftypefnx {} {[@var{V}, @var{lambda}, @var{W}] =} eig (@var{A}, @var{B})
 @deftypefnx {} {[@dots{}] =} eig (@var{A}, @var{balanceOption})
 @deftypefnx {} {[@dots{}] =} eig (@var{A}, @var{B}, @var{algorithm})
 @deftypefnx {} {[@dots{}] =} eig (@dots{}, @var{eigvalOption})
-Compute the right eigenvalues(V) and optionally the eigenvectors(lambda) and
-the left eigenvalues(W) of a matrix or a pair of matrices.
+Compute the eigenvalues (@var{lambda}) and optionally the right eigenvectors
+(@var{V}) and the left eigenvectors (@var{W}) of a matrix or a pair of
+matrices.
 
 The flag @var{balanceOption} can be one of:
 
 @table @asis
 @item @qcode{"balance"}
 Preliminary balancing is on. (default)
 
 @item @qcode{"nobalance"}
